pkg/controller/inferenceservice/reconcilers/knative/service_reconciler.go

```diff
@@ -84,51 +67,47 @@ func (r *ServiceReconciler) reconcileCanary(isvc *v1alpha2.InferenceService) err
 }
 
 func (r *ServiceReconciler) reconcileComponent(isvc *v1alpha2.InferenceService, component constants.InferenceServiceComponent, isCanary bool) error {
+	endpointSpec := &isvc.Spec.Default
+	serviceName := constants.DefaultServiceName(isvc.Name, component)
+	propagateStatus := isvc.Status.PropagateDefaultStatus
```

Comment:
Can we make clear this is a function? PropogateStatusFn?

Reply:
done